docs/source/release/v6.3.0/ModelFitting/ModelFitting.rst

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Im not sure if this location will work long term. However, we should keep these separate from the main muon release notes to prevent us forgetting anything we have changed and to hide them from users (since it is not visible to them yet)

Reply:
Just had a talk with Martyn and we can add it to the muon release notes as a comment https://stackoverflow.com/questions/4783814/how-to-comment-a-string-in-restructured-text and to make things easier we can just dump them all at the end of the file